dictionary.txt

```diff
@@ -136,3 +136,4 @@ versioning
 webmail
 wifi
 www.gitbook.com
+Theys
```

Comment:
:) I was wondering about that test fail. Thanks for identifying the issue.

Reply:
Since the rest of the file is alphabetical, I think this addition would be nice to also be alphabetical.

Reply:
Opps, I was trying to suggest the edit, and when I got the the screen to save the "suggestion", and it was like commit? and I went for it.